### PR TITLE
docs: update README.md

### PR DESCRIPTION
**Pull request by noahnu**
Add related links & license section. Decrease size of logo and remove poorly formatted badge table.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,3 +96,11 @@ This section is automatically generated via tagging the all-contributors bot in
 ```text
 @all-contributors please add <username> for <contribution type>
 ```
+
+## Related Links
+
+- JavaScript snapshot testing: https://github.com/facebook/jest
```
Comment (iamogbz) [inline README.md]:
```suggestion
- JavaScript snapshot testing: [jest](https://github.com/facebook/jest)
```

File: README.md [hunk 2]
```diff
@@ -1,18 +1,16 @@
 # syrupy
 
-<span><img align="right" width="200" height="200" src="https://user-images.githubusercontent.com/2528959/69500147-85d71400-0ec6-11ea-867a-277881278e57.png" alt="Logo"></span>
+<img align="center" width="100px" height="100" src="https://user-images.githubusercontent.com/2528959/69500147-85d71400-0ec6-11ea-867a-277881278e57.png" alt="Logo">
```
Comment (iamogbz) [inline README.md]: This does not seem to align center when rendered.

Comment (noahnu) [reply README.md]: removed redundant alignment

File: README.md [hunk 3]
```diff
@@ -1,18 +1,16 @@
 # syrupy
 
-<span><img align="right" width="200" height="200" src="https://user-images.githubusercontent.com/2528959/69500147-85d71400-0ec6-11ea-867a-277881278e57.png" alt="Logo"></span>
+<img width="100px" height="100px" src="https://user-images.githubusercontent.com/2528959/69500147-85d71400-0ec6-11ea-867a-277881278e57.png" alt="Logo">
```
Comment (iamogbz) [inline README.md]: I just think it looks better right aligned

Screen Shot 2019-12-26 at 11 10 37 PM

```suggestion
<img align="right" width="100px" height="100px" src="https://user-images.githubusercontent.com/2528959/69500147-85d71400-0ec6-11ea-867a-277881278e57.png" alt="Logo">
```

**Pull request merged**